docs/getting_started.md

```diff
@@ -152,3 +152,11 @@ There are several things you need to check before making your putting your appli
 
 [docker]: https://docs.docker.com/engine/installation/
 [docker-compose]: https://docs.docker.com/compose/install/
+
+## Contributing
+
+We always welcome new contributors of all levels to the project. If you are not confident enough with Ruby or web development you can look for [issues](https://github.com/decidim/decidim/issues) labeled `good first issue` to start contibuting and learning the internals of the project by doing easy jobs.
+
+We also have a [developer's reference](docs/development_guide.md) that will help you getting started with your environment and our daily commands, routines, etc.
+
+Finally, you can also find other ways of helping us on our [contribution guide](CONTRIBUTING.md).
```

Comment:
The file path for `CONTRIBUTING.md` is not correct, it should be `../CONTRIBUTING.md`